py-polars/polars/dataframe/frame.py

```diff
@@ -7796,16 +7796,18 @@ def pivot(
         Parameters
         ----------
         on
-            Name of the column(s) whose values will be used as the header of the output
+            The column(s) whose values will be used as the new columns of the output
             DataFrame.
         index
-            One or multiple keys to group by. If None, all remaining columns not specified
-            on `on` and `values` will be used. At least one of `index` and `values` must
-            be specified.
+            One or multiple columns to group by. These are the columns which will remain from the input
```

Comment:
I adapted from your suggestion but didn't take it exactly. I don't like saying it "form"s the output rows because it takes both the index and the values to form the output rows so I don't know if that might be confusing for someone. I did incorporate the unique combination making up the rows though.

Reply:
Yeah I agree with that, thanks! Pivot explanations are so darn hard to word...